### ISSUE TITLE
Error: SyntaxError: Invalid or unexpected token

### ISSUE THREAD

**Issue by zhengchencai** [r-causal/ggdag#156]
Hello,

Thanks for developing this nice tool. I was using ggdag with digitty for 2 years. Just updated ggdag today, and even for the simplest example, it will return me "Error: SyntaxError: Invalid or unexpected token". Do you know how to fix this, it seems also related to the update of dagitty to 0.3.4. 

I also tried to install it from github but got an error 

```
devtools::install_github("r-causal/ggdag")
Using GitHub PAT from the git credential store.
Error: Failed to install 'unknown package' from GitHub:
  HTTP error 401.
  Bad credentials

  Rate limit remaining: 54/60
  Rate limit reset at: 2024-04-24 13:57:58 UTC
```

I also tried to install the older version back, but got error

```
> install_version("ggdag", version = "0.2.8")
Downloading package from url: https://cran.rstudio.com//src/contrib/Archive/ggdag/ggdag_0.2.8.tar.gz
Error: Failed to install 'unknown package' from URL:
  HTTP error 401.
  Bad credentials

  Rate limit remaining: 52/60
  Rate limit reset at: 2024-04-24 13:57:58 UTC
```

When I just install it with install.packages(), I got the following message

```
  There is a binary version available but the source version is later:
      binary source needs_compilation
ggdag  0.2.8 0.2.12             FALSE
```

Thanks,
Zhengchen

**Comment by malcolmbarrett** [comment]
> I also tried to install it from github but got an error

This is a problem with your GitHub auth, not ggdag. See https://happygitwithr.com/https-pat

> When I just install it with install.packages(), I got the following message

 Enter yes when it asks you to install source. ggdag doesn't need a compiler so it won't take long. It would be good for you to try on the latest version.

**Comment by zhengchencai** [comment]
Thank you very much @malcolmbarrett. The example I tried is from the readme,

```
dag <- dagitty::dagitty("dag {
    y <- x <- z1 <- v -> z2 -> y
    z1 <- w1 <-> w2 -> z2
    x <- w1 -> y
    x <- w2 -> y
    x [exposure]
    y [outcome]
  }")
```

I did ```install.packages("dagitty", type = "binary")``` which installs the older version 0.3.1. Then everything works as usual. It is also the version I was using. 

I also updated ggdag to the latest version, which works fine. However, as long as I have the latest dagitty version, it will return me "Error: SyntaxError: Invalid or unexpected token". I guess this is not a ggdag issue, I have asked in dagitty github. 

Thanks a lot.
